plat/imx/common/imx_sip_handler.c

```diff
@@ -13,6 +13,12 @@
 #include <imx_sip_svc.h>
 #include <sci/sci.h>
 
+#ifdef PLAT_IMX8QM
```

Comment:
Is this actually working? The names of the defines are in lowercase: ```PLAT_imx8qm``` (you can see that if you build with ```V=```). This actually applies to other places in the port of this platform.

Reply:
I see, fair enough then, though it is a bit confusing.